Source/Libraries/Adapters/DynamicCalculator/EmailNotifier.cs

```diff
@@ -46,12 +46,14 @@ public class EmailNotifier : DynamicCalculator
         private const int DefaultFramesPerSecond = 30;
         private const double DefaultLagTime = 5.0D;
         private const double DefaultLeadTime = 5.0D;
+        private const bool DefaultSpamTriggerPrevention = false;
 
         // Fields
         private readonly Mail m_mailClient;
         private long m_expressionSuccesses;
         private long m_expressionFailures;
         private long m_totalEmailOperations;
+        private bool m_multiTriggerPrevention = false;
```

Comment:
The `MultiTriggerPrevention` property is an auto-property. By convention, if this property had a backing field it would be named `m_multiTriggerPrevention`. The `m_multiTriggerPrevention` field is instead being used a state that detects "when" multiple triggers have occurred. Without carefully looking, one might assume that the `m_multiTriggerPrevention` was a backing field, when it is not. I suggest renaming the field to something like `m_triggerDetected`. 

Also, assigning a field to its default value (`false`) is unnecessary.